packages/contracts-bedrock/src/L2/SuperchainWETH.sol

```diff
@@ -98,4 +119,55 @@ contract SuperchainWETH is WETH98, IERC7802, ISemver {
         return _interfaceId == type(IERC7802).interfaceId || _interfaceId == type(IERC20).interfaceId
             || _interfaceId == type(IERC165).interfaceId;
     }
+
+    /// @notice Sends ETH to some target address on another chain.
+    /// @param _to       Address to send tokens to.
+    /// @param _chainId  Chain ID of the destination chain.
+    function sendETH(address _to, uint256 _chainId) external payable returns (bytes32 msgHash_) {
+        if (_to == address(0)) revert ZeroAddress();
+
+        if (IL1Block(Predeploys.L1_BLOCK_ATTRIBUTES).isCustomGasToken()) {
+            revert NotCustomGasToken();
+        }
+
+        // NOTE: 'burn' will soon change to 'deposit'.
+        IETHLiquidity(Predeploys.ETH_LIQUIDITY).burn{ value: msg.value }();
+
+        msgHash_ = IL2ToL2CrossDomainMessenger(Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER).sendMessage({
+            _destination: _chainId,
+            _target: address(this),
+            _message: abi.encodeCall(this.relayETH, (msg.sender, _to, msg.value))
+        });
+
+        emit SendETH(msg.sender, _to, msg.value, _chainId);
+    }
+
+    /// @notice Relays ETH received from another chain.
+    /// @param _from       Address of the msg.sender of sendETH on the source chain.
+    /// @param _to         Address to relay ETH to.
+    /// @param _amount     Amount of ETH to relay.
+    function relayETH(address _from, address _to, uint256 _amount) external {
+        if (msg.sender != Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER) revert Unauthorized();
+
+        (address crossDomainMessageSender, uint256 source) =
+            IL2ToL2CrossDomainMessenger(Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER).crossDomainMessageContext();
+
+        if (crossDomainMessageSender != address(this)) revert InvalidCrossDomainSender();
+
+        if (IL1Block(Predeploys.L1_BLOCK_ATTRIBUTES).isCustomGasToken()) {
+            _mint(_to, _amount);
```

Comment:
do we want to add a comment explaining "why" are we minting WETH on customGasToken chains?
we could even link back to the relevant design-doc/spec

Reply:
done